src/assets/data/resources.json

```diff
@@ -65,6 +65,16 @@
             "design"
         ]
     },
+    {
+        "title": "Prepros",
+        "description": "Compiles your files, transpiles your JavaScript, reloads your browsers and makes it really easy to develop & test your websites so you can focus on making them perfect.",
+        "image": "https://prepros.io/img/icon.png",
```

Comment:
Next by default does not allow unknown image links. For safety's sake, we will continue with this approach. I released the URL user-images.githubusercontent.com, you can create an issue, attach the image to get a URL and dismiss an issue.